diff --git a/libtiff/CMakeLists.txt b/libtiff/CMakeLists.txt
--- a/libtiff/CMakeLists.txt
+++ b/libtiff/CMakeLists.txt
@@ -34,15 +34,15 @@ set(tiff_public_HEADERS
         tiff.h
         tiffio.h
         tiffvers.h
+        tif_dir.h
+        tiffiop.h
+        ${CMAKE_CURRENT_BINARY_DIR}/tif_config.h
         ${CMAKE_CURRENT_BINARY_DIR}/tiffconf.h)
 
 set(tiff_private_HEADERS
         t4.h
-        tif_dir.h
         tif_predict.h
-        tiffiop.h
-        uvcode.h
-        ${CMAKE_CURRENT_BINARY_DIR}/tif_config.h)
+        uvcode.h)
 
 
 add_library(tiff libtiff.def)
